content/docs/start/experiments.md

```diff
@@ -138,6 +160,55 @@ Experiment results have been applied to your workspace.
 
 <details>
 
+### ℹ️ More information about (Hyper)parameters and metrics
```

Comment:
Hopefully, we can condense this section and incorporate it with initializing the project, or directly before or after.

Reply:
The key points to me are that code should read parameters from a file and write metrics to a file. Not sure we need to go deeper than that.

Reply:
I simplified the section. Divided params and metrics. PTAL.